activity.go

```diff
@@ -2,44 +2,86 @@ package activity
 
 import (
 	"fmt"
+	"html/template"
 	"os"
 	"path"
 	"strings"
+	"time"
 
 	"github.com/jinzhu/gorm"
+	"github.com/qor/media_library"
 	"github.com/qor/qor"
 	"github.com/qor/qor/admin"
 	"github.com/qor/qor/audited"
+	"github.com/qor/qor/resource"
+	"github.com/qor/qor/utils"
+	"github.com/qor/qor/validations"
 )
 
 type QorActivity struct {
 	gorm.Model
-	Comment      string
+	Action       string
+	Content      string
```

Comment:
Content should give a default size, by default, the length is 255, maybe it is not enough. same to Note.